modules/organization/iam-roles-cloudscanner.tf

```diff
@@ -360,7 +360,7 @@ resource "google_organization_iam_binding" "cloudscanner_scaler_snapshot_deleter
 }
 
 resource "google_organization_iam_binding" "cloudscanner_sa_storage_object_reader_role_binding" {
-  count  = var.enable_cloudscanners ? 1 : 0
+  count  = (var.enable_cloudscanners && var.enable_dspm_scanning) ? 1 : 0
```

Comment:
should we update this line also? https://github.com/upwindsecurity/terraform-google-onboarding/blob/e28d95b64da4ec66430c3d0744d41397526a6498/modules/organization/iam-roles-cloudscanner.tf#L256